TehanovAnton/hometask-5/hometask

```diff
@@ -0,0 +1,73 @@
+1) select * from actor where last_name = 'Zellweger';
+
+2) select f.title, l.name
+	from film f inner join language l on f.language_id = l.language_id 
+	where l.name = 'English' or l.name = 'German' or l.name = 'Italian';
+
+3) select first_name || '-' || last_name
+	from actor
+	where first_name like 'W%' or last_name like '%h';
+
+4) select rental_duration, count(rental_duration) count
+	from film
+	group by rental_duration;
+
+5) select l.name as language, count(f.title)    
+	from language l inner join film f on l.language_id = f.language_id 
+	group by l.name;
```

Comment:
![image](https://user-images.githubusercontent.com/4544015/142903742-0d7766f7-e126-401d-8fc3-549bed7025c0.png)
And other languages?

Reply:
All films are in English. There are no films in other languages.